Bluepill-runner/Bluepill-runner/main.m

```diff
@@ -97,16 +99,20 @@ int main(int argc, char * argv[]) {
             exit(1);
         }
         [[BPStats sharedStats] endTimer:@"Loading App" withResult:@"INFO"];
+        // Always create simulator template(s)
```

Comment:
Please elaborate more in the comment... say that in order to read the tests we need to launch the app and inject code.